media/transcoder/api/Transcoder.Samples/CreateJobWithConcatenatedInputs.cs

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START transcoder_create_job_with_concatenated_inputs]
+
+using Google.Api.Gax.ResourceNames;
+using Google.Cloud.Video.Transcoder.V1;
+
+public class CreateJobWithConcatenatedInputsSample
+{
+    public Job CreateJobWithConcatenatedInputs(
+        string projectId, string location, string inputUri1, decimal startTimeOffset1, decimal endTimeOffset1, string inputUri2, decimal startTimeOffset2, decimal endTimeOffset2, string outputUri)
```

Comment:
What units are these offsets in? Seconds? I'd include that in the parameter name.
(It's also not clear to me that `decimal` is the right type here. For continuous quantities I'd usually suggest using `double` instead.)

Better yet, just accept a `TimeSpan` which is the more natural .NET way of representing a duration.

Reply:
Updated here and in other Transcoder samples